src/main/webapp/css/main.css

```diff
@@ -14,36 +14,53 @@
  * limitations under the License.
  */
 
+
 /* Prevent the page from becoming too wide. */
+
 body {
-  max-width: 100%;
+    max-width: 100%;
```

Comment:
Could we use two spaces, no tabs? This is based on the Airbnb ESLint config, for both CSS and JS files.

Reply:
I am using a auto indentation styling extension in visual studio called JS-CSS-HTML Formatter, maybe that is why some of the indentation of the files have changed..

Reply:
ok i just found another extension with eslint integration called prettier. Thanks for pointing out this issue!